test/atomsbase.jl

```diff
@@ -89,7 +95,6 @@ end
     atoms = @test_logs((:warn, r"Unitful quantity massdata is not yet"),
                        (:warn, r"Writing quantities of type Symbol"),
                        (:warn, r"Unitful quantity md is not yet"),
-                       (:warn, r"Mismatch between atomic numbers and atomic symbols"),
```

Comment:
This is because setting the `atomic_symbol` atprop now has no longer the effect of changing the atomic symbol as this is handled by the species. I'd suggest to still trigger this warning in the tests though by setting custom species and including a deuterium in there. This should be doable just by replacing `atomic_symbol=[ ... ]` by `species=ChemicalSpecies.([ .. ])`.